scripts/build-json/related-content.js

```diff
@@ -69,9 +69,16 @@ function buildSection(sectionSpec) {
  * Build a single related content object given its YAML file.
  * At the top level a related content object is an array of sections.
  */
+const relatedContentCache = {};
 function buildRelatedContent(specName) {
+  const cached = relatedContentCache[specName];
+  if (cached) {
```

Comment:
shouldn't that be `if (cached !== undefined)` or something? In case, the outcome of `spec.map(buildSection);` is something falsy.

Reply:
Makes sense @peterbe, updated to check for undefined. 👍